tasks/fix-a-bug.md

```diff
@@ -0,0 +1,32 @@
+#Labyrinth: Fix a bug
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Create or work on an  existing issue(Please take permission from the opener) to solve a bug in the site.
+  These are different aspects which you can cover:
+  - What problems it causes?
+  - What is needed to implement it
+  - How you propose an implementation
+- Get feedback from one maintainer (it is labeled as [bug](https://github.com/fossasia/labyrinth/labels/new%20feature))
+  or two other participants (they say this feature is good for the game).
```

Comment:
> they say this feature is good for the game

I would say, one is enough and change it to:

> they say this is a bug - this can be in the chat or in the issue